api/lib/application/error-manager.js

```diff
@@ -506,6 +500,14 @@ function _mapToHttpError(error) {
     return new HttpErrors.BadRequestError(error.message, 'SENDING_EMAIL_TO_INVALID_EMAIL_ADDRESS', error.meta);
   }
 
+  if (error instanceof DomainErrors.CertificationCandidateNotFoundError) {
+    return new HttpErrors.NotFoundError(error.message, error.code, error.detail);
```

Comment:
du coup on le retire lui c'est ça ? 

```suggestion
    return new HttpErrors.NotFoundError(error.message, error.code);
```